src/Reducers/userReducer.jsx

```diff
@@ -0,0 +1,59 @@
+export const Reducer = (state, action) => {
+  switch (action.type) {
+    case "Load_User":
```

Comment:
Someone told me that industry standard is to write the case (action.type) in capitals or uppercase. For ex. case "LOAD_USER", case "UPDATE_USER_PROFILE"

Reply:
Will change all reducer action to UpperCase.